dependencies.gradle

```diff
@@ -0,0 +1,28 @@
+ext.complexify = [
+    "minSdk"     : 21,
+    "compileSdk" : 30,
+    "targetSdk"  : 30,
+    "versionName": "3.0.0",
+    "versionCode": 3 * 100 * 100 + 0 * 100 + 0
+]
```

Comment:
Move this to config.gradle and rename or split into buildConfig and releaseConfig.